README.gcp.md

```diff
@@ -0,0 +1,127 @@
+# DDS - Build, test, and deploy using the Google Cloup Platform (GCP)
+
+For project info please see README.md
+
+# Connect to the cloud shell
+
+Using your favorite web browser, open a cloud shell terminal at <https://ssh.cloud.google.com/cloudshell>.
+You'll need a free Google account. These days, most people have one already.
+
+# Build and install the C++ library
+
+This repository contains the hackathon's fork of Bo Haglund's double-dummy solver code.
+
+```
+gcloud config set project online-bridge-hackathon-2020
+mkdir ~/bridge-hackathon/
+cd ~/bridge-hackathon/
+git clone https://github.com/suokko/dds.git
+cd dds/
+mkdir .build
+cd .build/
+cmake ..
+make
+```
+
+Optional, to make sure the library built correctly.
+It will take about six minutes to complete.
+```
+make check
+```
+
+Must be copied. A hard link is not possible and a symbolic link won't work:
+```
+sudo cp -f src/libdds.so.2 /usr/lib/
```

Comment:
Can you be more specific about _git rebase_? I'm new to git. I understand the problem, and I'd certainly like to know how to make it easier to review my changes.

Reply:
I'd love to have cmake do the install. I cannot at present. cmake installs the library in usr/local/lib/. By default, the cloud shell Python does not search there. There's surely a way to make this work.

Reply:
`git rebase` uses your remote tracking branch as base default. If you are tracking your push branch then you need to specify a commit (branches and tags are just commits with human readable name) which you want to use as a base. Most common case is that origin/master is the correct base.

`git rebase -i origin/master` 

This will then open your $EDITOR with list commits. You can see first column having all commits marked with `pick`. If you just save the file and quit then rebase will move to base commit and pick each commit in order which they were. This allows you to reorder commits by moving their line order in editor.

On top of reordering commits you can change `pick` to other documented operations like `edit` to make changes to any commit when rebase is done or `fixup` to  combine a commit to previous commit in the list.

Reply:
> I'd love to have cmake do the install. I cannot at present. cmake installs the library in usr/local/lib/. By default, the cloud shell Python does not search there. There's surely a way to make this work.

You can configure different prefix example would be
`cmake -DCMAKE_INSTALL_PREFIX=/usr ..`

Configure variables can be also edited later with `ccmake` tool or editing CMakeCache.txt with text editor.

Reply:
Done. I'll update the (rebased) pull shortly.